environment.sh

```diff
@@ -75,38 +76,18 @@ if [ $create_env == 'no' ]; then
     echo "Environment is older than one week old. Deleting and remaking environment...."
     conda remove -n $env_name --all -y
     create_env="yes"
+  else
+    echo "Environment created within the last week."
   fi
-fi
-
-if [ $create_env == 'yes' ]; then
-  # Create new environment
-  echo "Environment $env_name does not exist. Creating new environment $env_name..."
-  # Create conda environment
-  conda create -n $env_name python=3.11 -y
-fi
-
-# Activate environment
-conda activate $env_name
-if [ $create_env == 'yes' ]; then
-  # Install json parser
-  conda install jq -y
-  update_env='yes'
-else
-  update_env='no'
-fi
-
-# At this point we have either a new environment or an environment that is less than a week old.
-# We need to check if we need to reinstall packages
-if [ $update_env == 'no' ]; then
   # Check if environment was build before last modification to requirements file
   if [[ $creation_time < $requirements_modification_time ]]; then
     echo "Requirements file last modified after environment was created. Reinstalling packages..."
-    update_env="yes"
+    create_env="yes"
+  else
+    echo "Environment created after most recent change to requirements file."
   fi
-fi
-
-# Check if there has been an update to vivarium packages since last modification to requirements file
-if [ $update_env == 'no' ]; then
+  # Check if there has been an update to vivarium packages since last modification to requirements file
+  # or more reccent than environment creation
   grep @ $install_file 
```

Comment:
This should also go inside an else block following the check for the age of the requirements.txt